tvbwidgets/ui/pse_launcher_widget.py

```diff
@@ -88,8 +88,8 @@ def _prepare_launch(self, where="Local"):
         y_values = self.compute_params_values(self.param_2.value)
         file_name = self.verify_file_name()
         self.progress.min = 0
-        self.progress.max = len(x_values) * len(y_values) + 1
-        self.progress.value = 1
+        self.progress.max = len(x_values) * len(y_values)
+        self.update_progress(0)
```

Comment:
I would like to keep the progress estimation to `len(x_values) * len(y_values) + 1`, where each if the simulation is one step, and the preparation is also 1, so give at the very start to the user the sense that things are starting to move ;-)